src/simplewallet/simplewallet.cpp

```diff
@@ -4736,12 +4724,66 @@ bool simple_wallet::stake_all(const std::vector<std::string> &args_)
 
   cryptonote::account_public_address address = m_wallet->get_address();
 
-  std::vector<cryptonote::account_public_address> addresses = { address };
-  std::vector<uint32_t> shares = { STAKING_SHARES };
+  if (local_args.size() % 2 == 0 || local_args.size() < 5)
+  {
+    fail_msg_writer() << tr("Usage: stake_all [index=<N1>[,<N2>,...]] [priority] <address1> <shares1> [<address2> <shares2> [...]] <expiration timestamp> <service node pubkey> <signature>");
+    fail_msg_writer() << tr("");
+    fail_msg_writer() << tr("Prepare this command with the service node using:");
+    fail_msg_writer() << tr("");
+    fail_msg_writer() << tr("./lokid --prepare-registration <address> <shares> [<address2> <shares2> [...]]");
+    fail_msg_writer() << tr("");
+    fail_msg_writer() << tr("This command must be run from the daemon that will be acting as a service node");
+    return true;
+  }
```

Comment:
A single stakeholder stake_all command would be
stake_all (address) (shares) which is < 5 arguments so stake_all fails.

Reply:
stake_all address shares timestamp pubkey signature

Reply:
> It's clear this will change for the multi stake so, though the index parsing/priority isn't in should be fine.

I don't understand this sentence.

Reply:
I didn't review this part because it'll change for supporting multi stake.